unix/main.c

```diff
@@ -416,7 +416,7 @@ int main(int argc, char **argv) {
             } else if (strcmp(argv[a], "-v") == 0) {
                 mp_verbose_flag++;
             } else if (strncmp(argv[a], "-O", 2) == 0) {
-                if (isdigit(argv[a][2])) {
+                if (isdigit((byte)argv[a][2])) {
```

Comment:
That is strange. The error is about the array subscript, so that would be ```a``` or ```2``` on that line, being of type char. But both are int? And the error goes away by making what seems like an unrelated change? (though it is indeed the case that isdigit should not be passed signed char)